mne/coreg.py

```diff
@@ -155,26 +148,25 @@ def create_default_subject(mne_root=None, fs_home=None, update=False,
             "subject folder." % ('fsaverage', subjects_dir))
 
     # make sure mne files exist
-    mne_fname = os.path.join(mne_root, 'share', 'mne', 'mne_analyze',
-                             'fsaverage', 'fsaverage-%s.fif')
+    mne_fname = os.path.join(os.path.dirname(__file__), 'data', 'fsaverage',
+                             'fsaverage-%s.fif')
     mne_files = ('fiducials', 'head', 'inner_skull-bem', 'trans')
     for name in mne_files:
-        fname = mne_fname % name
-        if not os.path.isfile(fname):
+        if not os.path.isfile(mne_fname % name):
             raise IOError("MNE fsaverage incomplete: %s file not found at "
-                          "%s" % (name, fname))
+                          "%s" % (name, mne_fname % name))
```

Comment:
You shouldn't need this check anymore. This should only happen if we make some packaging error, and in that case the resulting `IOError` will be clear enough